source/fe/fe_q_base.cc

```diff
@@ -456,18 +456,18 @@ FE_Q_Base<PolynomialType,dim,spacedim>::initialize (const std::vector<Point<1> >
     this->poly_space.set_numbering(renumber);
   }
 
-  // finally fill in support points on cell and face
-  initialize_unit_support_points (points);
-  initialize_unit_face_support_points (points);
-
-  // reinit constraints
-  initialize_constraints (points);
+  // Finally fill in support points on cell and face and initialize
+  // constraints. All of this can happen in parallel
+  Threads::TaskGroup<> tasks;
+  tasks += Threads::new_task ([&]() { initialize_unit_support_points (points); });
```

Comment:
Seems you like the new possibility to have one line brace enclosed blocks. :smiley:

Reply:
I am surprised that astyle is happy with that one.

Reply:
Recently, we allowed having brace-enclosed one-liners. The main idea at that point was to allow for initializing variables, but we could only do this by allowing brace-enclosed one-liners in general.
Actually, I am not opposed to using it like that.

Reply:
It's not a brace-enclosed initializer here, though. Or am I missing something in the discussion?

The alternative here would have been to write
```
  Threads::new_task (&FE_Q_Base<dim,spacedim>::initialize_unit_support_points,
                                  this,
                                  points);
```
but that just seems more complicated for no real gain.

Reply:
> It's not a brace-enclosed initializer here, though. Or am I missing something in the discussion?

No, it's not a brace-enclosed initializer. What I wanted to say is that `astyle` can't differentiate between a brace-enclosed initializer and anything else that is written within braces in one line.
We just don't get more granularity using `astyle`.
I am not sure if we should (try to) avoid using this style for anything but brace-enclosed initializers but in this case I am totally fine with it.